### ISSUE TITLE
Accessibility Findings from Submit Evidence Review

### ISSUE THREAD

**Issue by brea11y** [department-of-veterans-affairs/va-mobile-app#9471]
During the accessibility audit of the updated submit evidence flow on August 27th, a couple of accessibility issues were discovered. None of the issues are considered "launch blockers". I would consider them to all be improvements that should be addressed as soon as we can.

I have added the issues below in the order that they should be fixed based on their severity.

- [ ] In the file request area, the headings are duplicated. I would remove the one within the card / box or change one of them to be unique / different so they are not duplicates.
<details><summary>Examples of duplicate headings</summary>
<img width="369" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBemF2Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--f74ad288bde1f59d2d91a85b74e5440300cba559/IMG_0026.PNG" alt="IMG_0026.PNG" />

<img width="369" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBem12Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--72585b502c14bd8d622857e336a078141b3ace9b/IMG_0025.PNG" alt="IMG_0025.PNG" />

<img width="369" src="https://api.zenhub.com/attachedFiles/eyJfcmFpbHMiOnsibWVzc2FnZSI6IkJBaHBBenF2Qmc9PSIsImV4cCI6bnVsbCwicHVyIjoiYmxvYl9pZCJ9fQ==--dd1d0779602f44b615e9c641314f6727b4adcfd1/IMG_0024.PNG" alt="IMG_0024.PNG" />
 </details> 

- [ ] Claim screen > Alert > "Review file requests" button. The screenreader only copy says "Navigates you to the find upload page". I would update this to "Navigates you to the file requests screen (or page)".
- [ ] File requests > [items that need additional information / files]. The screenreader only copy says "Go to the file request details page to upload file". I would update this to "Navigates you to the file upload screen (or page)".

**Comment by Sparowhawk** [comment]
In slack thread https://adhoc.slack.com/archives/C067S0FRP0C/p1726495408040769?thread_ts=1726494362.376019&cid=C067S0FRP0C

It was determined that duplicate titles/headings would be addressed globally in a separate ticket. so this only handles the hints on the 2 buttons

**Comment by ala-yna** [comment]
@brea11y / @Sparowhawk, what are the next steps here?

**Comment by TKDickson** [comment]
No further testing required in develop; closing
